Experimental/DedicatedGameServer/Assets/Scripts/Runtime/Game/GameplayObjects/SwitchedDoor.cs

```diff
@@ -198,8 +198,8 @@ void OnDoorCanBeOpenedChanged(bool couldBeOpened, bool canBeOpened)
             }
         }
 
-        [ServerRpc(RequireOwnership = false)]
-        void OnServerOpenServerRpc()
+        [Rpc(SendTo.Server)]
```

Comment:
this should be for the host too (same goes for other RPCs: the original implementation of ServerRpc and ClientRpc executed on Host + target (client in the 1st case, server in the 2nd))

Reply:
I just tested with client-hosted game with `Rpc(SendTo.Server)`:
* Host invokes ServerRpc, it's also invoked locally on host
* Client invokes ServerRpc, it's only invoked on the host.

Let me know if I'm missing something, since this would seem to work in either a client-hosted game, and similarly for a dedicated server topology.

Reply:
all good then!